### DISCUSSION TITLE
Pass location names to TSML (having swapped from "Legacy" UI

### DISCUSSION THREAD

**Discussion by k8fish3r** [code4recovery/12-step-meeting-list#1460] [Q&A] [answered]
Hi,

I used to use pass users to 12SML map system by using the URL structure of the legacy UI.

E.g. https://codauk.org/meetings/?tsml-day=any&tsml-distance=10&tsml-mode=location&tsml-query=EpsomUK&tsml-view=map#

Wondering what the URL format is for the latest instantiation of TSML please? Love the new front end - but sorely missing tsml-query parameter

Kate F

**Comment by joshreisner** [comment, marked as answer]
hi kate, yes the url parameters changed a bit with TSML UI. you can achieve this now with:

https://codauk.org/meetings/?mode=location&distance=10&view=map&search=epsom+uk

there is an improvement action here for TSML UI to keep the `search` param updated with location searches: https://github.com/code4recovery/tsml-ui/issues/383

also i created a new improvement action for TSML to automatically redirect you when the TSML UI appearance is selected and old parameters are detected: https://github.com/code4recovery/12-step-meeting-list/issues/1461

**Reply by k8fish3r** [reply]
Many thanks Josh I appreciate your support.

Ty

Kate F :)
